src/include/wiredtiger.in

```diff
@@ -2256,6 +2256,31 @@ struct __wt_connection {
 	 */
 	int __F(set_timestamp)(
 	    WT_CONNECTION *connection, const char *config);
+
+	/*!
+	 * Rollback in-memory non-logged state to an earlier point in time.
+	 *
+	 * This method requires that the application use timestamps as a
+	 * timestamp is how the rollback point is defined. This method requires
+	 * that there are no active cursor operations for the duration of
+	 * the call.
```

Comment:
In particular, it requires that a `stable_timestamp` be set via `WT_CONNECTION::set_timestamp`. before it is called.